source/features/unwrap-unnecessary-dropdowns.tsx

```diff
@@ -36,12 +36,44 @@ async function unwrapNotifications(): Promise<void | false> {
 	button.textContent = `Group by ${button.textContent.toLowerCase()}`;
 }
 
+function unwrapRerunActions(): void {
+	const menu = $('.PageHeader-actions action-menu');
```

Comment:
If you don't use observe(), this will run too early, especially on slow connection, and it won't find anything.

You can see that the other part of the feature uses elementReady for this.